flink-cdc-connect/flink-cdc-source-connectors/flink-cdc-base/src/test/java/org/apache/flink/cdc/connectors/base/source/assigner/state/PendingSplitsStateSerializerTest.java

```diff
@@ -152,7 +152,7 @@ public Offset createNoStoppingOffset() {
     private SchemalessSnapshotSplit constuctSchemalessSnapshotSplit() {
         return new SchemalessSnapshotSplit(
                 tableId,
-                "test",
+                tableId.toString() + ":0",
```

Comment:
Is is possible to make it `@VisibleForTesting` and call `generateSplitId`, to reduce potential discrepancy?